src/plugins/notebook/snapshot.js

```diff
@@ -108,7 +108,8 @@ export default class Snapshot {
         }
 
         return () => {
-            window.location.href = window.location.origin + url;
+            const path = window.location.href.split('#');
```

Comment:
`window.location.origin` gives only main domain address eg: 'https://XYZ....nasa.gov/' and not full path of domain eg "https://XYZ.....nasa.gov/path_name/"